distribution/docs/src/main/resources/_contents/configuring-from-the-admin-console-contents.adoc

```diff
@@ -5,7 +5,7 @@
 
 ===== Initial Configuration
 
-The first time the ${branding} {admin-portal} runs, the initial configuration steps appear.
+The first time the ${branding} ${admin-portal} runs, the initial configuration steps appear.
```

Comment:
shows up as "The first time the DDF ${admin-portal} runs, the initial configuration steps appear."